omigami/plotting.py

```diff
@@ -122,15 +142,16 @@ def _get_bonds_to_highlight(molecule: Mol, substructure: Mol) -> List[int]:
     # Author: Takayuki Serizawa
     # Original Source: https://iwatobipen.wordpress.com/2017/02/25/draw-molecule-with-atom-index-in-rdkit/
     @staticmethod
-    def _add_index_to_atoms(molecule: Mol) -> Mol:
+    def _add_index_to_atoms(molecule):
         for atom in molecule.GetAtoms():
             atom.SetAtomMapNum(atom.GetIdx())
         return molecule
 
+
     # Original Source: https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y
     @staticmethod
     def plot_classyfire_result(
-        spectra_matches: pd.DataFrame, color="g"
+            spectra_matches: pd.DataFrame, color="g"
```

Comment:
doesn't this one need the try catch as well?

Reply:
No, this one works without rdkit :)